src/middleware/navigation/is.secure.update.middleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { logger } from '../../utils/logger';
+import { SECURE_UPDATE_FILTER_URL } from '../../config';
+import { getApplicationData } from "../../utils/application.data";
+import { checkIsSecureUpdateDetailsEntered, NavigationErrorMessage } from './check.condition';
+
+export const isSecureUpdate = (req: Request, res: Response, next: NextFunction): void => {
```

Comment:
Do we need a new middleware for this or could we apply the existing to both registration & update?

Reply:
@AndreKramer7654 I have updated this after discussing with Jitin previously.

Reply:
It may be better to keep them separate but perhaps we could share checkIsSecure...Entered() and if we passed in the key for the item that we require to be there.